colabcode/code.py

```diff
@@ -40,3 +44,9 @@ def _run_code(self):
         ) as proc:
             for line in proc.stdout:
                 print(line, end="")
+                
+    def _install_packages(self):
+        for package in self.packages:
+            print(f'Installing {package}...')
```

Comment:
use logger.info instead of print here please

Reply:
yes adding it